### ISSUE TITLE
[Bug]: not injecting

### ISSUE THREAD

**Issue by chipper12345** [YimMenu/YimMenu#3657]
### Describe the bug

I have done the same thing every time to inject the menu into gta, however now it wont inject at all. I open process hacker 2, inject it into the game but nothing shows up, as in the menu isn't injecting or loading properly, any ideas why??

### Steps To Reproduce

1. Open process hacker and GTA.exe
2. Inject the dll into GTA.exe

### Expected Behavior

I expected the menu to open instead of not launching.

### Logs

I know this is bad to not include the logs but it opens windows player legacy and thats the only option, it wont let me use another app even if i had one?

### Store

Steam

### Language

English

### YimMenu Language

English (US)

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

**Comment by ghost** [comment]
you prob need to update ur dll

**Comment by chipper12345** [comment]
> you prob need to update ur dll

ive tried nightly and the one from the website and neither work, it normally works with both

**Comment by xenix4845** [comment]
That's right. Currently, when I try injection, the game is turning off with a crash. I'm sure there's a problem with the latest version.

**Comment by ghost** [comment]
it works perfectly for me, running it on windows 11 Epic edition
